massive-data-import/README.md

```diff
@@ -186,6 +192,9 @@ time docker-compose exec kuzzle node /scripts/loadMCreate.js
 ```
 
 On a laptop with a I5-7300U CPU @ 2.60 GHz, 16GiB of RAM and a SSD it takes approximatively 1 minutes to load 1 millions of document in Kuzzle with the Bulk Api method and approximatively 2 minutes with mCreate method.  
-This is faster than the cluster result because we don't have any latency.
+This is 40% faster than the cluster result because we don't have any latency.
```

Comment:
The `40% faster` part is confusing because of the mCreate method being `40% slower` than the bulk one (unfortunate coincidence)

Moreover, I don't know if that information is relevant. Once again, I don't think that providing numbers obtained on a laptop environment is helpful in any way.
But if you do insist on adding them, I'd change that sentence slightly, for instance: `(these numbers are higher than the cluster ones because there is no network latency)`